libs/pcre/Makefile

```diff
@@ -56,8 +57,8 @@ CONFIGURE_ARGS += \
 	--enable-unicode-properties \
 	--enable-pcre16 \
 	--with-match-limit-recursion=16000 \
-	--enable-cpp
 
+CONFIGURE_ARGS += $(if $(CONFIG_PACKAGE_libpcrecpp),--enable,--disable)-cpp
```

Comment:
Actually i believe you can replace the previous line with this. No need for the CONFIGURE_ARGS +=